omod/src/main/java/org/openmrs/module/shr/rest/web/controller/EncounterController.java

```diff
@@ -111,9 +107,14 @@ public Object postDocument(
 			if (log.isDebugEnabled()) {
 				log.debug("Request Response - " + error);
 			}
-			
 			response.setStatus(error.responseCode);
 			return error.response;
+		} catch (ContentHandlerException error){
```

Comment:
It's because saveContent method can throw these exceptions. The interface that includes this method is in org.openmrs.module.shr.contenthandler.api package.